### PR TITLE
Docs update

### PR DESCRIPTION
**Pull request by adscib**
Work in progress improving the documentation. So far I rearranged the structure and did the landing page.

### CODE REVIEW

File: docs/source/index.md [hunk 1]
```diff
@@ -8,43 +8,96 @@ hide-toc: true
...
 ```
 
-# Introduction
+[pypi-badge]: https://badge.fury.io/py/invertedai.svg
+[pypi-link]: https://pypi.org/project/invertedai/
+[colab-badge]: https://colab.research.google.com/assets/colab-badge.svg
+[colab-link]: https://colab.research.google.com/github/inverted-ai/invertedai-drive/blob/develop/examples/Colab-Demo.ipynb
 
-## Overview
+[![PyPI][pypi-badge]][pypi-link]
```
Comment (adscib) [inline docs/source/index.md:16]: Can we make a similar badge linking to our GitHub repo?

File: docs/source/index.md [hunk 2]
```diff
@@ -8,43 +8,96 @@ hide-toc: true
...
+
+The following pages will help you integrate the Inverted AI API with your own simulator.
+
 ```{toctree}
-:caption: INTEGRATIONS
 :maxdepth: 2
 
-
-carlasim
+userguide
```
Comment (adscib) [inline docs/source/index.md:97]: Is it somehow possible to link to the landing page from the left pane with some alias like "Quick start" or "Overview"?

Comment (AlirezaMorsali) [reply docs/source/index.md:97]: "self" would link to the landing page.

Comment (adscib) [reply docs/source/index.md:80]: done by adding `self` to toc

File: docs/source/index.md [hunk 3]
```diff
@@ -2,49 +2,85 @@
 hide-toc: true
 ---
 
-```{eval-rst}
-.. image:: ../images/banner.svg
-  :width: 600
-  :alt: Alternative text
-```
+![](../images/top_camera.gif)
```
Comment (adscib) [inline docs/source/index.md:5]: Frank suggested that we replace this with a row of three smaller gifs to make it look like a banner.

Comment (AlirezaMorsali) [reply docs/source/index.md:5]: Should we use existing gifs (the ones in the slides for example) or generate new ones?

Comment (adscib) [reply docs/source/index.md:5]: Only do this if you have easy access to such gifs already. If they need to be generated, we'll worry about that later.

**Pull request merged**